.github/workflows/deploy.yml

```
@@ -3,45 +3,57 @@
+      - name: Build the project
         env:
           NODE_ENV: production
           API_URL: ${{ secrets.API_URL }}
           CDN_URL: ${{ secrets.CDN_URL }}
-        run: |
-          npm ci
-          npm run build
+        run: npm run build
+
+      - name: Set up AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.4)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)